### PR TITLE
Add version to client instance

### PR DESCRIPTION
**Pull request by stubailo**
TODO:

- [ ] If this PR is a new feature, reference an issue where a consensus about the design was reached (not necessary for small changes)
- [ ] Make sure all of the significant new logic is covered by tests
- [ ] Rebase your changes on master so that they can be merged easily
- [ ] Make sure all tests and linter rules pass
- [ ] Update CHANGELOG.md with your change
- [ ] Add your name and email to the AUTHORS file (optional)
- [ ] If this was a change that affects the external API, update the docs and post a link to the PR in the discussion

### CODE REVIEW

**Comment by glasser** [comment]
It looks like you released this. Don't forget to merge.

**Comment by stubailo** [comment]
Gosh, thanks.

File: scripts/deploy.sh [hunk 1]
```diff
@@ -18,11 +18,13 @@ cd ./lib/src && cp -r ./ ../../npm/ && cd ../../
...
+  fs.writeFileSync('./npm/version.js', 'exports.version = \"' + package.version + '\"'); \
+  fs.writeFileSync('./npm/package.json', JSON.stringify(package, null, 2));"
 
 cp README.md npm/
 cp LICENSE npm/
 cp .npmignore npm/
 
 echo 'deploying to npm...'
-cd npm && npm publish
+cd npm && npm publish --tag test-version
```
Comment (helfer) [inline scripts/deploy.sh]: Did you maybe forget to remove this after testing?

Comment (stubailo) [inline scripts/deploy.sh]: omg I am literally the worst.

**Pull request merged**